oauthproxy.go

```diff
@@ -534,7 +543,7 @@ func (p *OAuthProxy) GetRedirect(req *http.Request) (redirect string, err error)
 	if req.Form.Get("rd") != "" {
 		redirect = req.Form.Get("rd")
 	}
-	if !p.IsValidRedirect(redirect) {
+	if redirect == "" || !p.IsValidRedirect(redirect) {
```

Comment:
Struggling to see why this has been changed, could you please explain

Reply:
Anything that doesn't explicitly set a redirection (such as a health check) winds up spewing a false-negative error message otherwise.

Reply:
Ack, I see the issue, raised #597 to solve that, seems a bit weird to be fixing a somewhat unrelated issue in this PR

Reply:
That's fine. I removed the check from this PR.